### PR TITLE
DHFPROD-8454: Reset to All Entities Hotfix

### PR DESCRIPTION
**Pull request by briantang**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- N/A Added Tests
- [x] Run UI tests
  

- ##### Reviewer:

- N/A Reviewed Tests
- N/A Added Release Notes

### CODE REVIEW

**Review by xnikhil08 — APPROVED** [review]

**Review by Sanjeevani19 — APPROVED** [review]

**Review by Sanjeevani19 — COMMENTED** [review]
@briantang Around 15 tests are failing on this PR, Please look into it

**Review by xnikhil08 — APPROVED** [review]

**Comment by briantang** [comment]
A couple e2e specs validating the explore table with faceting need to be skipped and completely overhauled with https://project.marklogic.com/jira/browse/DHFPROD-8455. This is due to a combination of my work + Ivey's entity specific facets, where all the existing faceting is now contained under the entity-specific panels. Many of these tests are still expecting to use the old options.

**Review by AkshayBajajML — APPROVED** [review]
Looks Good!!

**Review by Sanjeevani19 — CHANGES_REQUESTED** [review]
@briantang Ideally we shouldn't be skipping out all the test suites, I understand that these tests needs to be updated with the new faceting changes. There are two options here:
1. Updating all those test suites and make sure all the tests are running fine
2. Due to time constraint if we need some time to update the tests, then we have to all test those scenarios manually 
@ngodugu-marklogic  Any thoughts ? Both of you please work together to get this going

File: marklogic-data-hub-central/ui/e2e/cypress/integration/mastering/e2eMasteringflow.spec.tsx [hunk 1]
```diff
@@ -389,7 +389,7 @@ describe("Validate E2E Mastering Flow", () => {
     browsePage.getFacetItemCheckbox("collection", mergeStep).should("to.exist");
     cy.findByTestId("clear-sm-Patient-merged").should("to.exist");
   });
-  it("Explore other collections", () => {
+  it.skip("Explore other collections", () => {   //THIS FAILS UNTIL ENTITY SPECIFIC FACETS PR IS IN (DHFPROD-7950), needs to use entity specific panel facets instead entity properties panel
```
Comment (aniamy) [inline marklogic-data-hub-central/ui/e2e/cypress/integration/mastering/e2eMasteringflow.spec.tsx:392]: 7950 is already in develop

**Comment by briantang** [comment]
> @briantang Ideally we shouldn't be skipping out all the test suites, I understand that these tests needs to be updated with the new faceting changes. There are two options here:
> 
> 1. Updating all those test suites and make sure all the tests are running fine
> 2. Due to time constraint if we need some time to update the tests, then we have to all test those scenarios manually
>    @ngodugu-marklogic  Any thoughts ? Both of you please work together to get this going

I think the best approach would be to use manual testing for now. Basically having to rewrite all of the explore faceting will take 1-2 days. We also need to unblock the pipelines for other PRs

**Review by aniamy — APPROVED** [review]

**Review by Sanjeevani19 — APPROVED** [review]

**Pull request merged**